test/src/main/java/org/apache/accumulo/test/fate/zookeeper/ZooLockIT.java

```diff
@@ -306,6 +363,245 @@ public void testUnexpectedEvent() throws Exception {
 
   }
 
+  @Test(timeout = 60000)
+  public void testLockSerial() throws Exception {
+    String parent = "/zlretry";
+
+    ConnectedWatcher watcher = new ConnectedWatcher();
+    try (ZooKeeperWrapper zk1 = new ZooKeeperWrapper(getCluster().getZooKeepers(), 30000, watcher);
+        ZooKeeperWrapper zk2 = new ZooKeeperWrapper(getCluster().getZooKeepers(), 30000, watcher)) {
+
+      zk1.addAuthInfo("digest", "accumulo:secret".getBytes(UTF_8));
+      zk2.addAuthInfo("digest", "accumulo:secret".getBytes(UTF_8));
+
+      while (!watcher.isConnected()) {
+        Thread.sleep(200);
+      }
+
+      // Create the parent node
+      zk1.createOnce(parent, new byte[0], ZooDefs.Ids.OPEN_ACL_UNSAFE, CreateMode.PERSISTENT);
+
+      ZooReaderWriter zrw1 = new ZooReaderWriter(getCluster().getZooKeepers(), 30000, "secret") {
+        @Override
+        public ZooKeeper getZooKeeper() {
+          return zk1;
+        }
+      };
+
+      final RetryLockWatcher zlw1 = new RetryLockWatcher();
+      final String zlPrefix1 = "zlock#00000000-0000-0000-0000-AAAAAAAAAAAA#";
+      ZooLock zl1 = new ZooLock(zrw1, parent);
+      zl1.setVMLockPrefix(new Prefix(zlPrefix1));
```

Comment:
If each zoolock object created its own unique uuid, then could possibly have a getVMLockPrefix() instead of a set method.  The set method is fine, just a thought.

Reply:
I can change it depending on the discussion regarding the UUID being static.